src/PdfSmith/Program.cs

```diff
@@ -94,6 +97,17 @@
     options.AddDefaultProblemDetailsResponse();
 });
 
+builder.Services.AddOpenApiOperationParameters(options =>
+{
+    options.Parameters.Add(new()
+    {
+        Name = TimeZoneService.X_TIME_ZONE,
+        In = ParameterLocation.Header,
+        Required = true,
+        Schema = OpenApiSchemaHelper.CreateStringSchema()
+    });
+});
+
```

Comment:
After adding this parameter, you need to enable it as in the already mentioned example at https://github.com/marcominerva/TinyHelpers/blob/b14b4d1c279a6a97a66bed90108204d80dc776fe/samples/TinyHelpers.AspNetCore.Sample/Program.cs#L53.